src/main/java/org/embulk/input/zendesk/ZendeskInputPlugin.java

```diff
@@ -240,7 +240,7 @@ public TaskReport run(final TaskSource taskSource, final Schema schema, final in
     @Override
     public ConfigDiff guess(final ConfigSource config)
     {
-        config.set("columns", new ObjectMapper().createArrayNode());
```

Comment:
Same as `embulk-input-jira` -- You may want to remove `import com.fasterxml.jackson.databind.ObjectMapper;` as well.

Reply:
ObjectMapper still use in some different place of this class.